symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/RealTimeEventListener.java

```diff
@@ -3,9 +3,9 @@
 import com.symphony.bdk.gen.api.model.*;
 
 /**
- * Interface definition for a callback to be invoked when a datafeed event is received.
+ * Interface definition for a callback to be invoked when a real-time event is received.
```

Comment:
... "is received from the datafeed."

Reply:
You could also add a link to https://developers.symphony.com/restapi/docs/real-time-events using Javadoc annotation : 
```
/**
 * @see <a href="https://developers.symphony.com/restapi/docs/real-time-events">real-time-events</a>
 */
```